walnut/cmd/check.go

```diff
@@ -37,25 +38,18 @@ import (
 var checkCmd = &cobra.Command{
 	Use:   "check",
 	Short: "check the file passed against a specified schema.",
-	Long: `The check command validates the given file against a specified 
-schema. This file can be a task template, a workflow template or an O² 
-DPL Dump. Each of those have a schema provided to validate against. 
-
-Usage:
-  walnut check --format [format] [file]
-
-Example:
-  walnut check --format workflow readout-sftb.yaml
+	Long: `The check command validates the given file against a specified schema. This file can be a task template, 
+a workflow template or an O² DPL Dump. Each of those have a schema provided to validate against.
 
 Valid schemata formats:
   workflow  task  dpl_dump`,
 
 	Run: func(cmd *cobra.Command, args []string) {
-		// fmt.Println("check called with arg: " + strings.Join(args, " "))
-
 		format, _ := cmd.Flags().GetString("format")
-		for _, filename := range args {
-			file, err := ioutil.ReadFile(filename)
+		filename, _ := cmd.Flags().GetStringArray("filename")
```

Comment:
Why is `filename` a `StringArray`? Do you validate multiple DPL dumps per call? I don't remember if we discussed this.

Reply:
Yep! I thought it translated naturally into converting a batch of DPL Dumps into their equivalent templates. An example use case: `walnut convert -f dump1.json dump2.json dump3.json`. Right now there is no accommodation for separate folders per dump. This could go either:
- unified `tasks` and `workflows` folders for all input dumps
- separate folders where the parent is the name of the dump

And this works just as well for a single DPL Dump.

Reply:
In order to run a workflow in AliECS, one has to anyway commit the WFT and TTs to a Git repo, so in any case I'd prefer 1 target dir (with the usual `tasks` and `workflows` subdirectories) even with multiple DPL inputs.